lib_xua/src/core/xua_commands.h

```diff
@@ -24,10 +24,25 @@ enum
     CLOCK_COUNT
 };
 
-/* c_aud_ctl */
-#define SET_SAMPLE_FREQ         4
-#define SET_STREAM_FORMAT_OUT   8
-#define SET_STREAM_FORMAT_IN    9
+/* c_aud_ctl commands - These are propagated from EP0 through EP buffer to Decouple. */
+/* Note we avoid special channel tokens - see xs1b_user.h*/
+#define XUA_AUDCTL_NO_COMMAND   0 /* No command from decouple - carry on exchanging samples. This MUST be value zero */
+
+/* These commands will reach I2S and will cause the audioloop to break and restart*/
+#define SET_SAMPLE_FREQ         4 
+
+/* These commands only go as far as Decouple where they are translated to the next block */
+/* Note these need to be in order in a contiguous block due to the way the code handles them in decouple */
+/* When starting a stream, format information is also sent */
+#define SET_STREAM_INPUT_START  20
```

Comment:
prefer _XUA_<name> for internal defines, just to keep them unique. And the _ denotes not for user use.

Reply:
Good point. I was following the previous convention but makes sense to refactor whilst in there. I have done it for all commands